package.json

```diff
@@ -34,8 +34,8 @@
   },
   "homepage": "https://github.com/filipesilva/rollup-plugin-angular-inline#readme",
   "devDependencies": {
-    "@types/mocha": "^2.2.39",
-    "@types/node": "^7.0.5",
+    "@types/mocha": "^2.2.41",
+    "@types/node": "^7.0.33",
```

Comment:
Added these types to ensure that everything would work within a contained environment